setup.cfg

```
@@ -46,7 +46,7 @@ addopts = -v -rsxfE --durations=10 --color=yes
 filterwarnings =
     # From Cython-1753
     ignore:can't resolve:ImportWarning
-    error:::dask[.*]
+    #error:::dask[.*]
```

This needs to be uncommented

```suggestion
    error:::dask[.*]
```